help/deploy/complete-setup/configure-auth.md

```diff
@@ -3,75 +3,94 @@ title: "Configure authentication"
 sidebar_position: 1
 ---
 
-Out-of-the-box, Datagrok offers many authentication methods, including popular methods such as SSO and OAuth.
+Datagrok supports authentication using:
 
-To accommodate any enterprise needs, we can develop a customer-specific integration.
+* [Login-password](#login-password-authentication)
+* [Oauth](#oauth-authentication): 
+  * [Google](#google-authentication)
+  * [Facebook](#facebook-authentication)
+  * [GitHub](#github-authentication)
+* [OpenID](#openid-authentication)
+* [LDAP](#ldap-authentication)
 
-All methods can be enabled separately or combined.
+You can enable all authentication methods separately or combined.
 
-[Learn more about authentification](../govern/authentication.md).
+If supported authentication methods do not work for you, contact us on <info@datagrok.ai> or [our community](https://community.datagrok.ai/) and we will discuss options for your specific case.
 
-## Common options
+[Learn more about authentification](../../govern/authentication.md).
 
-If you want to display something on the login window, set the 'Prompt' option
-![Login prompt](login-prompt.png)
+## Common options
 
-Enable the 'Send Welcome Email' option to send welcome emails to new signed-up users.
+You can display a text on the login window, like greetings words etc.
```

Comment:
Purpose-location-action.

```suggestion
To display text on the login window, like greetings words, in **Settings** > **Users and Sessions** > General set *Prompt* option.
```

Reply:
But I still think it is an extra and should be removed from docs.

Reply:
Okay, so much fight for normal image size...